src/scss/variables.scss

```diff
@@ -4,3 +4,8 @@
 $border-radius: 0.25rem;
 $default-padding: 10px;
 $light-gray: $gray-200;
+
+// Shared style for the CodeView and DiffView components.
+$code-font-family: Consolas, Courier, monospace;
```

Comment:
This is not the same as what was in `CodeView/styles.module.scss`. Is this an improvement on that?